### PR TITLE
Added plaid library

### PR DESCRIPTION
**Pull request by blwinters**

### CODE REVIEW

File: package.json.j2 [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "q": "1.1.2", 
     "winston": "^2.2.0", 
     "winston-daily-rotate-file": "^1.0.1",
+    "plaid-node": "~1.0.10"
```
Comment (kotchaosu) [inline package.json.j2]: please add comma at the end of line

Comment (blwinters) [inline package.json.j2]: I've added the comma.

**Comment by kotchaosu** [comment]
hmm...

```
npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/plaid-node
npm ERR! 404 
npm ERR! 404  'plaid-node' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'nodejs-codebox'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

are you sure it's available?

**Comment by kotchaosu** [comment]
looks good - merge!

**Comment by kotchaosu** [comment]
@blwinters I will be available in about three hours

**Comment by blwinters** [comment]
@Zhebr Thanks! It is working for me.

**Pull request merged**